packages/polling-controller/src/PollingController.test.ts

```diff
@@ -30,14 +30,14 @@ describe('PollingController', () => {
       controller.startPollingByNetworkClientId('mainnet');
       jest.advanceTimersByTime(TICK_TIME);
       controller.stopAllPolling();
-      expect(controller.executePoll).toHaveBeenCalledTimes(1);
+      expect(controller._executePoll).toHaveBeenCalledTimes(1);
```

Comment:
In the same vein as not testing private code, it seems like a smell that we would test that this is called, if we truly want this to be a private method. Is there another way we can verify that a poll "iteration" was run?

Reply:
yeah I'll dig into how to rewrite these tests to achieve this.

Reply:
I could spy on a mocked call within the executePoll implementation?
```
const createExecutePollMock = () => {
  const executePollInternalCall = jest.fn();
  const executePoll = () => {
    executePollInternalCall();
  };
  return { executePoll, executePollInternalCall };
};
```
And then make assertions against the `executePollInternalCall` mock? Doesn't really add up to a different thing but in the context of testing the mixin itself I'm not thinking of a different way to achieve this?

Reply:
Ah, right, you're testing PollingController in this case by making a subclass. So in other words you control `_executePoll` in this case. Okay — this is probably the best way to do this then.